blocks/footer/footer.css

```diff
@@ -1,147 +1,352 @@
-}
+  footer .footer-links .icon-email > svg{
+    margin: 0.48rem auto;
+  }
 
-footer .footer-links .icon-email > svg{
-  margin: 0.48rem auto;
-}
+  footer .footer-links div:last-of-type > p:last-of-type {
+    display: flex;
+  }
 
-footer .footer-links .icon-linkedin > svg{
-  margin: 0.4rem auto;
-  width: 20px;
-  height: 20px;
-}
+  footer .footer-links .icon-linkedin > svg{
+    margin: 0.4rem auto;
+    width: 20px;
+    height: 20px;
+  }
 
-footer .footer-links .icon-twitter{
-  margin-left: 21px;
-}
+  footer .footer-links .icon-twitter{
+    margin-left: 21px;
+  }
 
-footer .footer-links .icon-twitter > svg{
-  margin: 0.5rem auto;
-  width: 20px;
-  height: 20px;
-}
+  footer .footer-links .icon-twitter > svg{
+    margin: 0.5rem auto;
+    width: 20px;
+    height: 20px;
+  }
 
-footer .footer-links .icon:hover svg{
-  filter: invert(100%);
-}
+  footer .footer-links .icon:hover svg{
+    filter: invert(100%);
+  }
 
-footer .footer-links .icon-footer-logo:hover {
-  background: none;
-}
+  footer .footer-links .icon-footer-logo:hover {
+    background: none;
+  }
 
-footer .footer-links .icon-footer-logo:hover svg{
-  filter: none;
-}
+  footer .footer-links .icon-footer-logo:hover svg{
+    filter: none;
+  }
 
-/* styles for the lower section links */
-footer .footer-links ul{
-  list-style: none;
-  margin: 0;
-  padding: 0;
-}
+  /* styles for the lower section links */
+  footer .footer-links ul{
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
 
-footer .footer-links ul li{
-  margin-bottom: 1.875rem;
-}
+  footer .footer-links ul li{
+    margin-bottom: 1.875rem;
+  }
 
-footer .footer-links ul li a{
-  color: var(--footer-font-color);
-  font-family: "Alliance Regular", sans-serif;
-  font-size: 14px;
-}
+  footer .footer-links ul li a{
+    color: var(--footer-font-color);
+    font-family: "Alliance Regular", sans-serif;
+    font-size: 14px;
+    text-decoration: none;
+  }
 
-footer .footer-links+ul{
-  list-style: none;
-  display: flex;
-  padding-top: 1rem;
-  padding-left: 0;
-  justify-content: space-between;
-}
+  footer .footer-links+ul>li a{
+    color:var(--footer-font-color);
+    text-decoration: none;
+  }
 
-footer .footer-links+ul li{
-  color:var(--footer-font-color);
-  font-size: 16px;
-}
+  footer .footer-links ul li a:hover {
+    text-decoration: underline;
+  }
 
-footer .footer-links+ul>li:first-of-type{
-  font-weight: 400;
-  width: 480px;
-}
+  footer .footer-links+ul{
+    list-style: none;
+    display: flex;
+    padding-top: 1rem;
+    padding-left: 0;
+    justify-content: space-between;
+  }
+
+  footer .footer-links+ul li{
+    color:var(--footer-font-color);
+    font-size: 16px;
+    flex: 1 1 0;
+  }
+
+ 
+
+  footer .footer-links+ul>li a:hover{
+    text-decoration: underline;
+  }
+
+  /* end styles for lower section */
+
+  /* Media queries */
+
+  /* || Begin Styles for screen sizes greater than or equal to 1120 px */
+
+  @media screen and (min-width: 1120px) {
```

Comment:
I think these media queries are maybe more complex than they need to be. Generally, I like to do the following:

* style mobile first. that is, the mobile styles (and styles that apply to all sizes aren't wrapped in a media query)
* layer on additional styles/overrides for tablet and desktop via min-width media queries

should end up with something like:

```
/* mobile and global styles */
@media (min-width: 700px) {
/* tablet styles */
}
@media (min-width: 900px) {
/* desktop styles */
}
@media (min-width: 1200px) {
/* wide screen styles */
}
```

You may be asking, why 700/900/1200?  good question. See https://www.freecodecamp.org/news/the-100-correct-way-to-do-css-breakpoints-88d6a5ba1862/

Reply:
100% agree and that is the learning that I am taking away from this POC. I should have started with 700 and grow into 1200px. Will try to refactor this code when i get an opportunity